packages/firebase_ml_vision/example/lib/main.dart

```diff
@@ -2,14 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
I updated this to include three demos. You will most likely have to pull in the latest changes.

Reply:
pulled in!